src/SwedbankPay/Api/Response.php

```diff
@@ -0,0 +1,240 @@
+     */
+    public function __isset($key)
+    {
+        return isset($this->object[$key]);
+    }
+
+    /**
+     * Unset
+     *
+     * @param $key
+     */
+    public function __unset($key)
+    {
+        unset($this->object[$key]);
+    }
+
+    /**
+     * Implementation of \ArrayAccess::offsetSet()
+     *
+     * @param string $offset
+     * @param mixed $value
+     * @return void
+     * @link http://www.php.net/manual/en/arrayaccess.offsetset.php
+     * @SuppressWarnings(PHPMD.ElseExpression)
+     */
+    public function offsetSet($offset, $value)
+    {
+        if (is_null($offset)) {
+            $this->object[] = $value;
+        } else {
+            $this->object[$offset] = $value;
+        }
+    }
+
+    /**
+     * Implementation of \ArrayAccess::offsetExists()
+     *
+     * @param string $offset
+     * @return bool
+     * @link http://www.php.net/manual/en/arrayaccess.offsetexists.php
+     */
+    public function offsetExists($offset)
+    {
+        return isset($this->object[$offset]);
+    }
+
+    /**
+     * Implementation of \ArrayAccess::offsetUnset()
+     *
+     * @param string $offset
+     * @return void
+     * @link http://www.php.net/manual/en/arrayaccess.offsetunset.php
+     */
+    public function offsetUnset($offset)
+    {
+        unset($this->object[$offset]);
+    }
+
+    /**
+     * Implementation of \ArrayAccess::offsetGet()
+     *
+     * @param string $offset
+     * @return mixed
+     * @link http://www.php.net/manual/en/arrayaccess.offsetget.php
+     */
+    public function offsetGet($offset)
+    {
+        return isset($this->object[$offset]) ? $this->object[$offset] : null;
+    }
+
+    /**
+     * Set/Get attribute wrapper
+     *
+     * @param $method
+     * @param $arguments
+     *
+     * @return mixed
+     * @throws Exception
+     */
+    public function __call($method, $arguments)
+    {
+        $key = lcfirst(mb_substr($method, 3, mb_strlen($method, 'UTF-8'), 'UTF-8'));
+        switch (mb_substr($method, 0, 3, 'UTF-8')) {
+            case 'get':
+                return $this->__get($key);
+            case 'set':
+                return $this->__set($key, $arguments[0]);
+            case 'uns':
+                return $this->__unset($key);
+            case 'has':
+                return $this->__isset($key);
+        }
+
+        throw new Exception(sprintf('Invalid method %s::%s', get_class($this), $method));
+    }
+
+    /**
+     * Extract operation value from operations list
+     *
+     * @param string $rel
+     * @param bool   $single
+     *
+     * @return bool|string|array
+     * @SuppressWarnings(PHPMD.BooleanArgumentFlag)
+     * @SuppressWarnings(PHPMD.UnusedLocalVariable)
+     */
+    public function getOperationByRel($rel, $single = true)
+    {
+        $operations = isset($this->object['operations']) ? $this->object['operations'] : [];
+        $operation  = array_filter($operations, function ($value, $key) use ($rel) {
```

Comment:
Codacy found an issue: [Avoid unused parameters such as '$key'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=6212562)

Reply:
@shahriar-vdc Check that please